sql-migrations/init_orders_db.sql

```diff
@@ -0,0 +1,17 @@
+-- +migrate Up
+-- +migrate StatementBegin
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS orders_db (
+    id uuid PRIMARY KEY,
+    user_id uuid NOT NULL,
+    label varchar NOT NULL,
+    created_at varchar NOT NULL
```

Comment:
created_at should be time-typed, not varchar.

Reply:
Done. Proto file fixed